plpm-persistence/src/main/java/cz/muni/fi/pa165/plpm/dao/GymDaoImpl.java

```diff
@@ -0,0 +1,59 @@
+package cz.muni.fi.pa165.plpm.dao;
+
+import cz.muni.fi.pa165.plpm.entity.Gym;
+import cz.muni.fi.pa165.plpm.entity.Trainer;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+import java.util.List;
+
+/**
+ * Implementation of GymDao
+ *
+ * @author Karolína Kolouchová
+ */
```

Comment:
DaoImpl class should be marked @ Repository

Reply:
Added